.github/workflows/rust.yml

```diff
@@ -57,123 +54,39 @@ jobs:
         os:  [ubuntu-latest]
         rust-channel: ['stable', 'nightly']
         rust-target:
-          - 'armv7-linux-androideabi'
-          - 'aarch64-linux-android'
-          - 'i686-linux-android'
-          - 'x86_64-linux-android'
+          - { triple: 'armv7-linux-androideabi', abi: armeabi-v7a }
+          - { triple: 'aarch64-linux-android', abi: arm64-v8a }
+          - { triple: 'i686-linux-android', abi: x86 }
+          - { triple: 'x86_64-linux-android', abi: x86_64 }
```

Comment:
I saw you created an issue about the `abi` mess, hope it'll be resolved soon.

Reply:
Yeah, it's a bit of a weird quirk with cargo ndk